byron/chain/executable-spec/src/Cardano/Spec/Consensus/Block.hs

```diff
@@ -21,11 +23,25 @@ class BlockHeader h where
   bhIssuer :: h -> VKey
   -- | Slot for which this block is issued
   bhSlot :: h -> Slot
+  -- | UTxO hash
+  bhUtxoHash :: h -> Hash
+  -- | Delegation hash
+  bhDlgHash :: h -> Hash
+  -- | Update payload hash
+  bhUpdHash :: h -> Hash
 
 
 class BlockBody bb where
   -- | Delegation certificates.
   bbCerts :: bb -> [DCert]
+  -- | UTxO payload
+  bbUtxo :: bb -> UTxO TxId
```

Comment:
Yeah, so this should probably be a list of `TxWits`

Reply:
Fixed.